byterun/obj.c

```diff
@@ -42,6 +42,7 @@ CAMLprim value caml_static_resize(value blk, value new_size)
   return (value) caml_stat_resize((char *) blk, (asize_t) Long_val(new_size));
 }
 
+// unused since GPR#427
```

Comment:
This should maybe a /* ... */ comment.  (Not sure exactly what our standards requirements are here, but it's what's used everywhere else)